scriptmodules/emulators/ags.sh

```diff
@@ -11,45 +11,79 @@
 
 rp_module_id="ags"
 rp_module_desc="Adventure Game Studio - Adventure game engine"
-rp_module_help="ROM Extension: .exe\n\nCopy your Adventure Game Studio roms to $romdir/ags"
+rp_module_help="ROM Extension: .exe\n\nCopy your Adventure Game Studio roms to $romdir/ags/<game>/"
 rp_module_licence="OTHER https://raw.githubusercontent.com/adventuregamestudio/ags/master/License.txt"
-rp_module_repo="git https://github.com/adventuregamestudio/ags.git release-3.5.1"
+rp_module_repo="git https://github.com/adventuregamestudio/ags.git master"
 rp_module_section="opt"
 rp_module_flags="!mali"
 
 function depends_ags() {
-    getDepends xorg pkg-config libaldmb1-dev libfreetype6-dev libtheora-dev libvorbis-dev libogg-dev liballegro4-dev
+    getDepends pkg-config libaldmb1-dev libfreetype6-dev libtheora-dev libvorbis-dev libogg-dev liballegro4-dev
 }
 
 function sources_ags() {
     gitPullOrClone
 }
 
 function build_ags() {
+    cmake -DCMAKE_INSTALL_PREFIX="$md_inst" \
+        -DAGS_USE_LOCAL_ALL_LIBRARIES=ON \
+        -DAGS_USE_LOCAL_SDL2_SOUND=OFF \
+        -DCMAKE_BUILD_TYPE=Release
     make -C Engine clean
     make -C Engine
-    md_ret_require="$md_build/Engine/ags"
+    md_ret_require="$md_build/ags"
 }
 
 function install_ags() {
-    make -C Engine PREFIX="$md_inst" install
+    make -C Engine install
+    # install Eawpatches GUS patch set (see: http://liballeg.org/digmid.html)
+    download "http://www.eglebbk.dds.nl/program/download/digmid.dat" - | bzcat >"$md_inst/bin/patches.dat"
```

Comment:
Just to note - moving this from the configure stage to the install stage means it will be packaged with our pre-built binaries rather than installed after. Not an issue, just noting.